### PR TITLE
website config and initial content

### PR DESCRIPTION
**Pull request by sag1v**
as per #48

### CODE REVIEW

File: website/pages/en/index.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    <img src={props.img_src} alt="Project Logo" />
+  </div>
+)
+
+const ProjectTitle = () => (
+  <h2 className="projectTitle">
+    {siteConfig.title}
+    <small>
+      <MarkdownBlock>
+        Official React bindings for [Redux](https://github.com/reduxjs/redux)
```
Comment (markerikson) [inline website/pages/en/index.js]: Need to change this heading.

File: website/pages/en/index.js [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+class HomeSplash extends React.Component {
+  render() {
+    const language = this.props.language || ''
+    return (
+      <SplashContainer>
+        <div className="inner">
+          <ProjectTitle />
+          <PromoSection>
+            <Button href={docUrl('introduction/quick-start', language)}>
```
Comment (markerikson) [inline website/pages/en/index.js]: We'll need a `docs/introduction/quick-start.md` page added

File: website/pages/en/index.js [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+}
+
+const Installation = () => (
+  <div
+    className="productShowcaseSection paddingBottom"
+    style={{ textAlign: 'center' }}
+  >
+    <h2>Installation</h2>
+    <MarkdownBlock>
+      Redux starter kit requires **Redux 4.0.0 or later.**
```
Comment (markerikson) [inline website/pages/en/index.js]: This package already depends on Redux directly, so this line doesn't seem necessary.

File: website/sidebars.json [hunk 4]
```diff
@@ -0,0 +1,20 @@
+{
+  "docs": {
+    "Introduction": [
+      "introduction/quick-start",
+      "introduction/basic-tutorial",
+      "introduction/why-use-redux-starter-kit"
+    ],
+    "Using redux-starter-kit": [
+      "using-redux-starter-kit/connect-mapstate",
```
Comment (markerikson) [inline website/sidebars.json]: These lines should be removed

File: website/sidebars.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
+  "docs": {
+    "Introduction": [
+      "introduction/quick-start",
```
Comment (markerikson) [inline website/sidebars.json]: We'll need to create some actual Markdown content, and replace these lines with the IDs from those files.

File: website/sidebars.json [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+      "introduction/basic-tutorial",
+      "introduction/why-use-redux-starter-kit"
+    ],
+    "Using redux-starter-kit": [
+      "using-redux-starter-kit/connect-mapstate",
+      "using-redux-starter-kit/connect-mapdispatch"
+    ],
+    "API Reference": [
+      "api",
+      "api/provider"
```
Comment (markerikson) [inline website/sidebars.json]: Also need to replace the API page references

File: website/siteConfig.js [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+  organizationName: 'reduxjs',
+  // For top-level user or org sites, the organization is still the same.
+  // e.g., for the https://JoelMarcey.github.io site, it would be set like...
+  //   organizationName: 'JoelMarcey'
+
+  // For no header links in the top nav bar -> headerLinks: [],
+  headerLinks: [
+    { doc: 'introduction/quick-start', label: 'Quick Start' },
+    {
+      doc: 'using-redux-starter-kit/connect-mapstate',
```
Comment (markerikson) [inline website/siteConfig.js]: Should point to whatever our actual first Markdown page is

File: website/pages/en/index.js (hunk 1 above)
Comment (sag1v) [reply website/pages/en/index.js]: Official starter kit for [Redux] ?

Comment (markerikson) [reply website/pages/en/index.js]: Let's go with:

> A simple set of tools to make using Redux easier

File: website/sidebars.json (hunk 5 above)
Comment (sag1v) [reply website/sidebars.json]: Where do i get this content from, `src/README.md` for now?

File: website/sidebars.json (hunk 6 above)
Comment (sag1v) [reply website/sidebars.json]: with `configureStore`, `getDefaultMiddleware`, `createReducer` etc.. ?

File: website/sidebars.json (hunk 5 above)
Comment (markerikson) [reply website/sidebars.json]: Yeah, that's what I suggested in my earlier comments.

File: website/siteConfig.js (hunk 7 above)
Comment (sag1v) [reply website/siteConfig.js]: so basically we just have 1 now,  `quick-start.md`. so remove all header links and leave only quick-start?

**Comment by markerikson** [comment]
Awright, this should be good enough for now.  Merging!

**Pull request merged**